src/pages/palanbyulChoi/Main/CommentList.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './CommentList.scss';
+import Comment from './Comment';
+class CommentList extends Component {
+  render() {
+    const { commentList, commentLoveClassName, onDelete } = this.props;
+    return (
+      <div className="comment_box">
+        {commentList.map(comment => {
+          return (
+            <div className="comment" key={comment.id}>
+              <Comment
+                userName={comment.userName}
+                comment={comment.comment}
+                commentLoveClassName={commentLoveClassName}
+                commentList={commentList}
+                onDelete={onDelete}
+                commentId={comment.id}
+              />
+            </div>
+          );
+        })}
+      </div>
+    );
+  }
+}
+
+export default CommentList;
```

Comment:
- import 순서 수정해 주시고,
- 줄 띄우기도 해주세요!
```suggestion
import React, { Component } from 'react';
import Comment from './Comment';
import './CommentList.scss';

class CommentList extends Component {
  render() {
    const { commentList, commentLoveClassName, onDelete } = this.props;
    return (
      <div className="comment_box">
        {commentList.map(comment => {
          return (
            <div className="comment" key={comment.id}>
              <Comment
                userName={comment.userName}
                comment={comment.comment}
                commentLoveClassName={commentLoveClassName}
                commentList={commentList}
                onDelete={onDelete}
                commentId={comment.id}
              />
            </div>
          );
        })}
      </div>
    );
  }
}

export default CommentList;
```

Reply:
넵 !